tests/lists.test.ts

```diff
@@ -1,15 +1,47 @@
 import { PyInt, PyBool, PyNone, PyObj } from "../utils";
 import { assert, asserts, assertPrint } from "./utils.test";
 
-describe("run", () => {
-  // assert('add', "2 + 3", PyInt(2 + 3));
-  // assert('Lists Declaration','items : [int] = None\nitems', PyNone())
-  // var source = `
-  //     items : [int] = None
-  //     n : int = 10
-  //     items = [1, 2]
-  //     n = items[0] + items[1]
-  //     n
-  // `
-  // assert('Lists Access', source,  PyInt(1 + 2))
+describe("LIST TEST", () => {
+  var source = `
+        items : [int] = None
+        items = [1, 2, 3]
+    `;
+
+  assert("Assign List To Variable (Program 2)", source, PyNone());
+
+  var source = `
+    items : [int] = None
+    items = [1, 2, 3] + [4, 5, 6]
```

Comment:
Would like to see a version of this test where the two lists are stored in variables, then concatenated.